salt/_states/metalk8s_cordon.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+def _node_unschedulable(name, value, **kwargs):
```

Comment:
I know it's not your code, but let's be a tad more explicit with function names (to avoid confusion with the module function `node_unschedulable` that only retrieves the flag value):

```suggestion
def _set_node_unschedulable(name, value, **kwargs):
```